### PR TITLE
change type of rtype to commonv1.ReplicaType 

### PR DESCRIPTION
**Pull request by MartinForReal**
related: #134

### CODE REVIEW

**Review by gaocegege — COMMENTED** [review]
/assign @terrytangyuan @zw0610 @carmark @Jeffwan

**Comment by carmark** [comment]
Did you do `gofmt`?

**Comment by MartinForReal** [comment]
> Did you do `gofmt`?

Please refer to https://travis-ci.com/github/kubeflow/common/builds/226788990

File: go.mod [hunk 1]
```diff
@@ -14,30 +15,9 @@ require (
 	volcano.sh/apis v1.2.0-k8s1.19.6
 )
 
-
```
Comment (terrytangyuan) [inline go.mod:17]: Where did this change come from?

Comment (MartinForReal) [reply go.mod:17]: Since we move to volcano.sh/api and these statements are no longer needed. Should I  move these changes to another change set?

Comment (terrytangyuan) [reply go.mod:17]: Yes separate PRs would be better.

Comment (MartinForReal) [reply go.mod]: OK. I've amended the commit.

File: go.sum [hunk 2]
```diff
@@ -1152,165 +443,64 @@ google.golang.org/protobuf v1.23.0/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2
...
 sigs.k8s.io/yaml v1.2.0 h1:kr/MCeFWJWTwyaHoR9c8EjH9OumOmoF9YGiZd7lFm/Q=
 sigs.k8s.io/yaml v1.2.0/go.mod h1:yfXDCHCao9+ENCvLSE62v9VSji2MKu5jeNfTrofGhJc=
-sourcegraph.com/sqs/pbtypes v0.0.0-20180604144634-d3ebe8f20ae4 h1:JPJh2pk3+X4lXAkZIk2RuE/7/FoK9maXw+TNPJhVS/c=
-sourcegraph.com/sqs/pbtypes v0.0.0-20180604144634-d3ebe8f20ae4/go.mod h1:ketZ/q3QxT9HOBeFhu6RdvsftgpsbFHBF5Cas6cDKZ0=
-vbom.ml/util v0.0.0-20160121211510-db5cfe13f5cc h1:MksmcCZQWAQJCTA5T0jgI/0sJ51AVm4Z41MrmfczEoc=
-vbom.ml/util v0.0.0-20160121211510-db5cfe13f5cc/go.mod h1:so/NYdZXCz+E3ZpW0uAoCj6uzU2+8OWDFv/HxUSs7kI=
 volcano.sh/apis v1.2.0-k8s1.19.6 h1:ddSHBrxHOVpGVSYQC/e696tDmCQ2dPm/Adg7aTQdOPI=
 volcano.sh/apis v1.2.0-k8s1.19.6/go.mod h1:UaeJ/s5Hyd+ZhFLc+Kw9YlgM8gRZ/5OzXqHa0yKOoXY=
-volcano.sh/volcano v1.2.0 h1:HW8tG4ZBvR+HaQMaeS9TuebKmzBohemKSiqibHNUKlw=
-volcano.sh/volcano v1.2.0/go.mod h1:zddAnaLKfnKMYkyMbdhlb8J3HwGeqvryeLl1tux/G4M=
```
Comment (terrytangyuan) [inline go.sum:1316]: Revert changes in go.sum too?

**Review by terrytangyuan — COMMENTED** [review]
LGTM. Others PTAL.

File: go.sum (hunk 2 above)
Comment (MartinForReal) [reply go.sum]: Sorry, another pr is submitted. Thanks! #136

**Comment by Jeffwan** [comment]
https://github.com/kubeflow/common/blob/eb55ddefdea6d4034af9f4fe66dd202ff3af400e/pkg/apis/common/v1/types.go#L151

Nice catch.  `ReplicaType` is string and we may forget to make them consistent. This will be a user facing change and operators will need to make corresponding change after upgrade. 

/lgtm

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**